libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1465,7 +1480,10 @@ void IRGeneratorForStatements::endVisit(MemberAccess const& _memberAccess)
 					break;
 				case FunctionType::Kind::Internal:
 					if (auto const* function = dynamic_cast<FunctionDefinition const*>(_memberAccess.annotation().referencedDeclaration))
+					{
 						define(_memberAccess) << to_string(function->id()) << "\n";
+						m_context.internalFunctionCalled(_memberAccess);
```

Comment:
Should this rather be `internalFunctionAccessed`?

Reply:
Right. I'll change it.

Reply:
Done.